### PR TITLE
LPS-177619 Remove Stream usages for module: commerce-theme-minium-impl

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:commerce

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/helper/CommerceThemeMiniumHttpHelper.java [hunk 1]
```diff
@@ -128,34 +127,31 @@ private String _getPortletURL(
...
-				null
-			);
+		if (ListUtil.isEmpty(layouts)) {
+			return StringPool.BLANK;
+		}
 
-			if (friendlyURLLayout != null) {
-				return _portal.getLayoutURL(friendlyURLLayout, themeDisplay);
+		for (Layout layout : layouts) {
+			if (Objects.equals(layout.getFriendlyURL(), friendlyURL)) {
```
Comment (dantewang) [inline modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/helper/CommerceThemeMiniumHttpHelper.java:135]: @julschong Looks good, just here, the looped item usually goes second.

**Comment by dantewang** [comment]
@julschong Just a minor change, please resend.

**Pull request closed without merging**